store/src/java/com/zimbra/cs/service/mail/SaveDraft.java

```diff
@@ -176,6 +177,7 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
                 try {
                     mSession = JMSession.getSmtpSession(mbox.getAccount());
                     if (null != mailAddress) {
+                        mSession.getProperties().setProperty(MAIL_AUTOSEND_TIME, String.valueOf(autoSendTime));
```

Comment:
This line should be added ouside this if(). Sent from can be null but in that case too mail is being sent to recepients and in that case autoSendTime should be set .